jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -20,7 +20,7 @@
 
     public static SchemaMetadata readRemoteMetadata(final String connectionString, final String user,
```

Comment:
Introduce parameter object.

Reply:
Leave that to a refactoring branch, because it is used in a lot of places and has downsides